.github/workflows/main.yml

```diff
@@ -34,15 +34,25 @@ jobs:
     - name: Test
       if: matrix.os != 'ubuntu-latest'
       run: |
+        dotnet sln neo.sln remove ./tests/Neo.Plugins.Storage.Tests/Neo.Plugins.Storage.Tests.csproj
         dotnet test
     - name: Test for coverall
       if: matrix.os == 'ubuntu-latest'
       run: |
+        sudo apt-get --assume-yes install libleveldb-dev librocksdb-dev
```

Comment:
`if: matrix.os != 'ubuntu-latest'` it's also required for `dotnet test` we need to do that always

Reply:
I removed the `Storage` tests from `windows` and `macos` this can be `permanent` or `temporarily`, up to you. Problem is that we would have to add `libleveldb` binary to this repo.

Reply:
@shargon I think what I'm going to do is do what i was doing in #3018 with `leveldb` building to fix it for `windows` and `macos` along with `arm64`.

Reply:
We need to test the storage

Reply:
@shargon what is a solution for `libleveldb` binaries in repo?

Reply:
This can be discussed in a different pr, this should be only a move

Reply:
Agree and I'll issue to for a note to fix it.